sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md

```diff
@@ -263,6 +263,9 @@ The valid values are:
 - WebPreviewableFiles
 - ServerRenderedFilesOnly
 
+> [!NOTE]
```

Comment:
[!NOTE] should not be used on cmdlet's articles as it can't be rendered in Get-Help output. Please replace with regular note.